packages/perseus/src/jquery.mobile.vmouse.js

```diff
@@ -21,6 +21,7 @@
 //
 // The current version exposes the following virtual events to jQuery bind methods:
 // "vmouseover vmousedown vmousemove vmouseup vclick vmouseout vmousecancel"
+import $ from "jquery";
```

Comment:
We were lucky that webapp exposes `$` and `jQuery` as globals.  Now that we're explicitly import jquery here we don't have to worry about things accidentally breaking in the future if we decide to stop setting those globals.